app/bundles/CampaignBundle/Views/Campaign/details.html.twig

```diff
@@ -0,0 +1,206 @@
+{% extends 'MauticCoreBundle:Default:content.html.twig' %}
+
+{% block headerTitle %}{{ campaign.name }}{% endblock %}
+
+{% block actions %}
+  {{- include('MauticCoreBundle:Helper:page_actions.html.twig', {
+    'item': campaign,
+    'templateButtons': {
+        'edit': securityHasEntityAccess(
+            permissions['campaign:campaigns:editown'],
+            permissions['campaign:campaigns:editother'],
+            campaign.createdBy
+        ),
+        'clone': permissions['campaign:campaigns:create'],
+        'delete': securityHasEntityAccess(
+            permissions['campaign:campaigns:deleteown'],
+            permissions['campaign:campaigns:deleteother'],
+            campaign.createdBy
+        ),
+        'close': securityHasEntityAccess(
+            permissions['campaign:campaigns:viewown'],
+            permissions['campaign:campaigns:viewother'],
+            campaign.createdBy
+        ),
+    },
+    'routeBase': 'campaign',
+  }) -}}
+{% endblock %}
+
+{% block publishStatus %}
+  {{ include('MauticCoreBundle:Helper:publishstatus_badge.html.twig', {'entity': campaign}) }}
+{% endblock %}
+
+{% block content %}
+{% set firstTab = 'preview' %}
```

Comment:
When viewing the PHP template, it doesn't seem like this could ever be anything but this.